File: sdk/elixir/lib/microsandbox.ex
defmodule Microsandbox do
  @moduledoc """
  A minimal SDK for the Microsandbox project.
  """

  @doc """
  Returns a greeting message for the given name.

  ## Examples

      iex> Microsandbox.greet("World")
      "Hello, World! Welcome to Microsandbox!"

  """
  @spec greet(String.t()) :: String.t()
  def greet(name) do
    message = "Hello, #{name}! Welcome to Microsandbox!"
    IO.puts(message)
    message
  end
end

File: sdk/julia/src/Microsandbox.jl
module Microsandbox

"""
    greet(name::String)

Returns a greeting message for the given name.

# Examples
```julia
julia> greet("World")
"Hello, World! Welcome to Microsandbox!"
```
"""
function greet(name::String)
    message = "Hello, $name! Welcome to Microsandbox!"
    println(message)
    return message
end

export greet

end # module
